### PR TITLE
Add Important Notice About the RIAA

### PR DESCRIPTION
**Pull request by polarisfm**
youtube-dl -x https://youtu.be/SnLB8wysMbY

### CODE REVIEW

**Comment by sickcodes** [comment]
LGTM

**Review by sickcodes — APPROVED** [review]

**Review by oofdere — APPROVED** [review]

**Comment by notviri** [comment]
hello, based department?

**Review by cybrefox — APPROVED** [review]

**Review by notviri — APPROVED** [review]

**Review by rbnkrnn — APPROVED** [review]

**Review by Sven65 — APPROVED** [review]

**Review by KeepSOBP — APPROVED** [review]

File: 2020/10/2020-10-24-Fuck-the-RIAA.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
+October 23, 2020
+
+GitHub
+
+Dear Sir or Madam:
```
Comment (bee-san) [inline 2020/10/2020-10-24-Fuck-the-RIAA.md]: Per #8129 let's change this to "To Whom It May Concern:"

**Review by penn5 — APPROVED** [review]

**Review by purringChaos — APPROVED** [review]
lgtm

**Review by 9ary — APPROVED** [review]

**Review by rottencandy — APPROVED** [review]

**Review by RektWithStyle — APPROVED** [review]

**Review by takarushi — APPROVED** [review]

File: 2020/10/2020-10-24-Fuck-the-RIAA.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+
+I am contacting you on behalf of the Free Software community and its members.
+
+Fuck the M-P-double-A comin' straight out the underground
+A young pirate got it bad cause I'm down,
+Loadin' DVDs like a motherfuckin' fiend,
+Bring my camera to the movies and I put em on BT,
+Back all that stolen content up on DVD-ROMs,
+'Cuz my tip's been piracy since I dropped out my Mom's,
+And just because I share my MP3s,
```
Comment (resynth1943) [inline 2020/10/2020-10-24-Fuck-the-RIAA.md]: Isn't MP3 a proprietary format 👁️

File: 2020/10/2020-10-24-Fuck-the-RIAA.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+They got the government comin' after me,
+Instead of suin' kids why don't you step on up,
+And release a couple albums that don't completely suck,
+Stop puttin DRM onto audio CDs,
+That don't make it any harder to steal your MP3s,
```
Comment (resynth1943) [inline 2020/10/2020-10-24-Fuck-the-RIAA.md]: Isn't MP3 a proprietary format eye

File: 2020/10/2020-10-24-Fuck-the-RIAA.md [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+Fuck the suits behind the BSA
+And fuck 'em all for the DMCA
+Fuck the M-P-double-A
+Fuck the R-I-double-A
+Fuck the suits behind the BSA
+And fuck 'em all for the DMCA
+
+Robot pirates, we get our shit for free,
+Parental Advisory you'll never fuckin' see,
+Been a couple years since I seen an FBI warning,
```
Comment (resynth1943) [inline 2020/10/2020-10-24-Fuck-the-RIAA.md]: FBI is nonfree software

File: 2020/10/2020-10-24-Fuck-the-RIAA.md (hunk 2 above)
Comment (JoelZimmerman) [reply 2020/10/2020-10-24-Fuck-the-RIAA.md]: Yeah, MP3 should be changed to Vorbis or maybe even FLAC.

**Review by JoelZimmerman — CHANGES_REQUESTED** [review]

File: 2020/10/2020-10-24-Fuck-the-RIAA.md (hunk 2 above)
Comment (polarisfm) [reply 2020/10/2020-10-24-Fuck-the-RIAA.md]: Changed.

File: 2020/10/2020-10-24-Fuck-the-RIAA.md (hunk 3 above)
Comment (polarisfm) [reply 2020/10/2020-10-24-Fuck-the-RIAA.md]: Changed.

File: 2020/10/2020-10-24-Fuck-the-RIAA.md (hunk 4 above)
Comment (polarisfm) [reply 2020/10/2020-10-24-Fuck-the-RIAA.md]: This is why the FBI warning has been removed. Shit's mad boring.

**Review by l1ving — APPROVED** [review]

**Review by NickAcPT — APPROVED** [review]
LGTM!

**Review by chylex — APPROVED** [review]

**Review by Margen67 — APPROVED** [review]

**Review by soatok — APPROVED** [review]

**Review by EnderDev — APPROVED** [review]

**Review by EnderDev — APPROVED** [review]

**Review by yourboilokey — APPROVED** [review]

**Review by curlpipe — APPROVED** [review]

**Review by Bloxycube — APPROVED** [review]
Masterpiece

**Review by spikecodes — APPROVED** [review]

**Review by EnderDev — APPROVED** [review]

**Review by EnderDev — APPROVED** [review]

**Review by spikecodes — APPROVED** [review]

**Review by EnderDev — APPROVED** [review]

**Review by EnderDev — APPROVED** [review]

**Review by EnderDev — APPROVED** [review]

**Review by johnpaulbin — APPROVED** [review]

**Review by bigspeedfpv — APPROVED** [review]

**Review by curlpipe — APPROVED** [review]

**Review by bigspeedfpv — APPROVED** [review]

**Review by DumbMahreeo — APPROVED** [review]

**Review by snoozed-dev — APPROVED** [review]

**Review by williamd1k0 — APPROVED** [review]

**Review by d678h — APPROVED** [review]

**Review by d678h — APPROVED** [review]

**Review by d678h — APPROVED** [review]

**Review by database64128 — APPROVED** [review]

**Review by dst212 — APPROVED** [review]

**Review by EnderDev — APPROVED** [review]

**Review by EnderDev — APPROVED** [review]

**Review by EnderDev — APPROVED** [review]

**Review by EnderDev — APPROVED** [review]

**Review by Delta-Glitch05 — APPROVED** [review]

**Review by MM-coder — APPROVED** [review]

**Review by resynth1943 — APPROVED** [review]

**Review by gluons — APPROVED** [review]

**Review by Vixtron — APPROVED** [review]

**Review by awnumar — APPROVED** [review]

**Review by imachug — APPROVED** [review]
Whoever wrote this has a talent.

**Review by Technerder — APPROVED** [review]

**Review by d678h — APPROVED** [review]

**Review by aeiouaeiouaeiouaeiouaeiouaeiou — APPROVED** [review]

**Review by resynth1943 — APPROVED** [review]

**Review by kubapolish — APPROVED** [review]

**Review by geonizeli — APPROVED** [review]

**Comment by jessephus** [comment]
Hi :wave: We are closing this pull request because we [do not accept contributions](https://github.com/github/dmca/blob/master/CONTRIBUTING.md) to this repository. Feel free to keep the discussion going in this thread. Please just be mindful to keep the discussion respectful and consistent with [GitHub’s Community Guidelines](https://docs.github.com/github/site-policy/github-community-guidelines).

We understand that you may have questions about this takedown. Although we do not normally monitor comments posted to this repository, our team is reading through the comments and questions in this and other recent threads to see where we may be able to clarify our policies or answer some common questions. Thanks.

**Review by d678h — APPROVED** [review]

**Review by reloonfire — APPROVED** [review]

**Review by Toydotgame — COMMENTED** [review]
Yeah, this should work.

**Pull request closed without merging**